### ISSUE TITLE
Api for MT4?

### ISSUE THREAD

**Issue by alvinluo666** [tarikhagustia/laravel-mt5#4]
Do you have this package for MT4?

**Comment by tarikhagustia** [comment]
yes i have, but is a private. please email me @ agustia.tarikh150@gmail.com

**Comment by hmpmarketing** [comment]
Hi,

I see your PHP-mt5 repo was removed due to take down notice. do you think you can share with me?
